src/PostgREST/Statements.hs

```diff
@@ -202,7 +202,7 @@ dbSettingsStatement = H.Statement sql HE.noParams decodeSettings False
     sql = [q|
       with
       role_setting as (
-        select unnest(setconfig) as setting from pg_catalog.pg_db_role_setting where setrole = 'postgrest_test_authenticator'::regrole::oid
+        select unnest(setconfig) as setting from pg_catalog.pg_db_role_setting where setrole = current_user::regrole::oid
```

Comment:
I'm pretty sure I started the notify feature with `current_user`. I must have changed it when doing tests and then I squashed the commits locally plus force pushed :man_facepalming: .

Reply:
(did that here: https://github.com/PostgREST/postgrest/pull/1729#issuecomment-762938712)